### PR TITLE
Accuracy results for scale estimation algorithm in docs.

### PR DESCRIPTION
**Pull request by andreyanufr**
### Changes

Added scale estimation accuracy results to documentation.

### CODE REVIEW

File: docs/compression_algorithms/CompressWeights.md [hunk 1]
```diff
@@ -61,10 +61,11 @@ nncf_dataset = nncf.Dataset(data_source, transform_fn)
...
 ```
 
-- Accuracy of the 4-bit compressed models also can be improved by using AWQ algorithm over data-based mixed-precision algorithm. It is capable to equalize some subset of weights to minimize difference between
+- Accuracy of the 4-bit compressed models also can be improved by using AWQ algorithm or Scale Estimation algorithm over data-based mixed-precision algorithm. It is capable to equalize some subset of weights to minimize difference between
 original precision and 4-bit.
-Below is the example how to compress 80% of layers to 4-bit integer with a default data-based mixed precision algorithm and AWQ.
-It requires to set `awq` to `True` additionally to data-based mixed-precision algorithm.
+Below is the example how to compress 80% of layers to 4-bit integer with a default data-based mixed precision algorithm and AWQ with Scale Estimation.
+It requires to set `awq` to `True` and `scale_estimation` to `True` additionally to data-based mixed-precision algorithm.
+Both algorithms (AWQ and Scale Estimation) works independently.
```
Comment (MaximProshin) [inline docs/compression_algorithms/CompressWeights.md:68]: "Both algorithms (AWQ and Scale Estimation) works independently." - I think it's unclear and I would rephrase it. How about "Both algorithms, AWQ and Scale Estimation, can be enabled together or separately."?

File: docs/compression_algorithms/CompressWeights.md [hunk 2]
```diff
@@ -389,6 +395,90 @@ This modification applies only for patterns `MatMul-Multiply-MatMul` (for exampl
     </tr>
 </table>
 
+Here is the perplexity and accuracy with data-free and data-aware mixed-precision INT4-INT8 weight compression for different language models on the [lambada openai dataset](https://huggingface.co/datasets/EleutherAI/lambada_openai).
+`_scale` suffix refers to the data-aware mixed-precision with Scale Estimation algorithm.
+`r10` means that embeddings and lm_head have INT8 precision and all other linear layers have INT4 precision.
```
Comment (MaximProshin) [inline docs/compression_algorithms/CompressWeights.md:400]: r10 should be r100? fp should be fp16? I would also change the order to align with prev tables.

**Review by MaximProshin — CHANGES_REQUESTED** [review]

File: docs/compression_algorithms/CompressWeights.md (hunk 1 above)
Comment (andreyanufr) [reply docs/compression_algorithms/CompressWeights.md:68]: Done

File: docs/compression_algorithms/CompressWeights.md (hunk 2 above)
Comment (andreyanufr) [reply docs/compression_algorithms/CompressWeights.md:400]: Done

File: docs/compression_algorithms/CompressWeights.md (hunk 1 above)
Comment (MaximProshin) [reply docs/compression_algorithms/CompressWeights.md]: > Both algorithms

remove unnecessary dot at the end of 'Both algorithms, AWQ and Scale Estimation, can be enabled together or separately..'

**Review by MaximProshin — APPROVED** [review]
Minor typo above. The rest is ok. 
Please also map it to the release branch after the merge.

File: docs/compression_algorithms/CompressWeights.md (hunk 1 above)
Comment (andreyanufr) [reply docs/compression_algorithms/CompressWeights.md]: Done
